src/emm/emm_cli.py

```diff
@@ -96,9 +96,12 @@ def create_pull_request(self, title: Optional[str], body: Optional[str]) -> None
         :param body: Body for the pull request.
         """
         created_pull_requests = self.pull_request_service.create_pull_request(title, body)
-        print("Created the following pull requests:")
-        for pr in created_pull_requests:
-            print(f"- {pr.name}: {pr.link}")
+        if created_pull_requests:
+            print("Created the following pull requests:")
+            for pr in created_pull_requests:
+                print(f"- {pr.name}: {pr.link}")
+        else:
+            print("No pull requests are being created.")
```

Comment:
Can we print more details as why no pull requests are being created (something like no changes on the current branch)?

Reply:
This information is already being handled here.https://github.com/evergreen-ci/evg-module-manager/blob/891e5bf52041549609bfb51059ebfbae5a56831b/src/emm/services/pull_request_service.py#L157

For push changes to remote, the error is handled [here](https://github.com/evergreen-ci/evg-module-manager/blob/891e5bf52041549609bfb51059ebfbae5a56831b/src/emm/clients/git_proxy.py#L303).

Also, for errors while creating the pr, the github cli tool would print out the error information [here](https://github.com/evergreen-ci/evg-module-manager/blob/891e5bf52041549609bfb51059ebfbae5a56831b/src/emm/clients/github_service.py#L44) and [here](https://github.com/evergreen-ci/evg-module-manager/blob/891e5bf52041549609bfb51059ebfbae5a56831b/src/emm/clients/github_service.py#L56). Those errors would be printed while user issue the pull-request command, so I think we just need to let user know that no prs are being created if they don't have errors.

Reply:
> For push changes to remote, the error is handled [here](https://github.com/evergreen-ci/evg-module-manager/blob/891e5bf52041549609bfb51059ebfbae5a56831b/src/emm/clients/git_proxy.py#L303).

The link you included is to logging code. That will not be displayed by default. Logging is typically for digging into details about what is going on with the tool, it is generally not a very friendly way to interface with a user. 

I think including a phrase in the output here to indicate that no changes were found seems appropriate.